compiler/main/driver.cpp

```diff
@@ -329,6 +330,8 @@ static bool fPrintVersion = false;
 static bool fPrintChplHome = false;
 
 std::string llvmFlags;
+char llvmRemarksFilters[128];
```

Comment:
It's my own opinion that we should use `std::string` for such things. I don't know if that's hard to do with the arg parser, but these kind of fixed-length buffers are a bit fraught and I think with C++ we have better ways of handling a string.

Reply:
I changed it to a `std::string`, set by a function `setLLVMRemarksFilters` instead of letting the argument parser set it since it only handles C string